src/analysis/modals/ComputeModal/GcpComputeModal/GcpComputeModal.test.ts

```diff
@@ -403,6 +404,12 @@ describe('GcpComputeModal', () => {
         selectRuntimeImage(GcpComputeImageSection, runtime);
       });
 
+      await user.click(screen.getByLabelText('CPUs'));
+      const selectOption = await screen.findByText('2');
+      await user.click(selectOption);
+      const nextButton = await screen.findByText('Next');
+      await user.click(nextButton);
```

Comment:
There was actually a bug that these tests relied on to pass that was fixed.

Essentially, the `canUpdateRuntime` function compares existing and desired configs and if it finds a difference say 'yes you can update'. The `toolLabel` in the desired config was incorrectly computed, and was undefined, which signaled to the modal an update was allowed (hence update button `Enabled`). The test on line 369 now correctly reads update should be disabled when no changes have been made, and this subsequent test actually requires a field to be changed (CPUs) for `Update` to be enabled. 

![Pasted Graphic](https://github.com/user-attachments/assets/de7857e3-0e86-41af-93d4-b095007968b0)